<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PurchaseOrderItemEditor</class>
 <widget class="QDialog" name="PurchaseOrderItemEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>69</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tambahkan Produk</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="1,0,0,0,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Pilih &amp;Produk:</string>
       </property>
       <property name="buddy">
        <cstring>productComboBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="newProductButton">
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Tambah produk baru</string>
       </property>
       <property name="text">
        <string>Produk Baru</string>
       </property>
       <property name="icon">
        <iconset resource="app.qrc">
         <normaloff>:/icons/add</normaloff>:/icons/add</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&amp;Kwantitas:</string>
     </property>
     <property name="buddy">
      <cstring>quantitySpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Harga &amp;Beli (Rp.):</string>
     </property>
     <property name="buddy">
      <cstring>priceEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Harga &amp;Jual (Rp.):</string>
     </property>
     <property name="buddy">
      <cstring>priceEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="4" rowspan="2">
    <widget class="QPushButton" name="addButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="app.qrc">
       <normaloff>:/icons/add</normaloff>:/icons/add</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="productComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QSpinBox" name="quantitySpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999999</number>
       </property>
       <property name="stepType">
        <enum>QAbstractSpinBox::DefaultStepType</enum>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="uomLabel">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>sat</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <widget class="CurrencyEdit" name="costEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="CurrencyEdit" name="priceEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CurrencyEdit</class>
   <extends>QLineEdit</extends>
   <header>libs/widgets/currencyedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>newProductButton</tabstop>
  <tabstop>productComboBox</tabstop>
  <tabstop>quantitySpinBox</tabstop>
  <tabstop>priceEdit</tabstop>
  <tabstop>addButton</tabstop>
 </tabstops>
 <resources>
  <include location="app.qrc"/>
 </resources>
 <connections/>
</ui>
